specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5280,6 +5424,75 @@
       },
       "description": "A skill that can call a Web API endpoint, allowing you to extend a skillset by having it call your custom code."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Skill"
+        }
+      ],
+      "properties": {
+        "entitiesDefinitionUri": {
+          "type": "string",
+          "description": "The uri pointing to a hosted json or csv file containing which entities to lookup."
+        },
+        "defaultLanguageCode": {
+          "$ref": "#/definitions/CustomEntityLookupSkillLanguage",
+          "description": "A value indicating which language code to use. Default is en."
+        },
+        "inlineEntitiesDefinition": {
+          "$ref": "#/definitions/CustomEntityLookupEntityDefinition",
+          "description": "An inline definition of entities to lookup."
+        }
+      },
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/azure/search/cognitive-search-skill-custom-entity-lookup"
```

Comment:
This is a 404 right now. I'm assuming this will be online soon?

Reply:
kind of.. It'll be released when we release the skill, which may not be for several weeks

---
In reply to: [348711440](https://github.com/Azure/azure-rest-api-specs/pull/7812#discussion_r348711440) [](ancestors = 348711440)